src/ompl/robots.cpp

```diff
@@ -3200,17 +3201,46 @@ void compute_col_shape(Motion &m, dynobench::Model_robot &robot) {
       co->setTranslation(transform.translation());
       co->setRotation(transform.rotation());
       co->computeAABB();
+      // if(merged)
+      //   collision_objects_tmp.push_back(std::move(co));
+      // always take
       m.collision_objects.push_back(std::move(co));
     }
   }
-
+  if (merged) {
+    // get the merged AABB
+    fcl::AABB<double> aabb_merged;
+    for (auto &tmp_co : m.collision_objects) {
+      auto aabb_tmp = tmp_co->getAABB();
+      aabb_merged += aabb_tmp;
+    }
+    // extract min, max of the merged AABB
+    fcl::Vector3<double> &max_coords = aabb_merged.max_;
+    fcl::Vector3<double> &min_coords = aabb_merged.min_;
+    // create a Box with using min, max as boundaries
```

Comment:
Did we/quim always use double for FCL rather than float? Float should be faster and accurate enough.